plugin.go

```diff
@@ -64,12 +69,49 @@ func execPlugin(plugin, pluginOpts, remoteHost, remotePort, localHost, localPort
 	return nil
 }
 
-func fileExists(filename string) bool {
-	info, err := os.Stat(filename)
-	if os.IsNotExist(err) {
-		return false
+func findPath(file string) (string, error) {
+	skip := []string{"/", "./", "../"}
+
+	for _, p := range skip {
+		if strings.HasPrefix(file, p) {
+			err := findExecutable(file)
+			if err != nil {
+				return "", err
+			}
+			return filepath.Abs(file)
+		}
+	}
+
+	executable, err := os.Executable()
```

Comment:
There is no guarantee that the path is still pointing to the correct executable.

When running `go run`, it won't return a correct path to the working dir. Why not just use `./`?